src/protocolsupport/protocol/typeremapper/tile/TileEntitySkullRemapper.java

```diff
@@ -25,10 +26,11 @@ protected static byte getLegacyData(BlockData skull) {
 
 	protected void register(List<Entry<Consumer<NBTCompound>>> list, Material skull, int skulltype) {
 		for (BlockData blockdata : MaterialAPI.getBlockDataList(skull)) {
-			byte rotation = getLegacyData(blockdata);
+			float rotation = getLegacyData(blockdata);
 			list.add(new Entry<>(MaterialAPI.getBlockDataNetworkId(blockdata), nbt -> {
 				nbt.setTag("SkullType", new NBTByte((byte) skulltype));
-				nbt.setTag("Rot", new NBTByte(rotation));
+				nbt.setTag("Rotation", new NBTFloat((float) (rotation * 22.5))); //For PE Clients
+				nbt.setTag("Rot", new NBTByte((byte) rotation)); //For PC Clients
 			}));
```

Comment:
I am vaguely okay with this, if not 100%. Shev will need to have the final say.

My recommendation otherwise is to let the constructor take a bool isPE, store it and use it to select which format to use, and to make two separate calls to registers.

Sorry for being such an annoying reviewer. :) You did a great fix! It's just that it's also important to keep the clean code base.

Reply:
@magicus The TileEntitySkullRemapper class is doing both already. I might've interpreted what you said wrong though lmao, but I will add in the isPE bool